scripts/skills/perks/perk_rf_offhand_training.nut

```diff
@@ -76,23 +76,12 @@ this.perk_rf_offhand_training <- ::inherit("scripts/skills/skill", {
 				continue;
 			}
 
-			if (i.isItemType(::Const.Items.ItemType.Tool))
+			if (i.getStaminaModifier() <= -10)
 			{
-				validItemsCount++;
-				continue;
+				return null;
 			}
 
-			if (i.isItemType(::Const.Items.ItemType.Shield))
-			{
-				 if (i.getID().find("buckler") != null)
-				 {
-					 validItemsCount++;
-				 }
-				 else
-				 {
-					 return null;
-				 }
-			}
+			validItemsCount++;
```

Comment:
You might aswell `return 0` here to simplify the logic of this function

Reply:
No because we have to iterate over all the items to ensure that the previous condition i.e. `i.getStaminaModifier() <= -10` isn't being applied to any item.